tests/BearSerializationTest.py

```diff
@@ -0,0 +1,40 @@
+import itertools
+import json
+import os
+import unittest
+
+from pyprint.NullPrinter import NullPrinter
+
+from coalib.bears.BEAR_KIND import BEAR_KIND
+from coalib.collecting.Collectors import collect_bears
+from coalib.output.JSONEncoder import create_json_encoder
+from coalib.output.printers.LogPrinter import LogPrinter
+
+
+class BearSerializationTest(unittest.TestCase):
+    """
+    Test to make sure all bears are serializable.
+    """
+
+    def setUp(self):
+        printer = LogPrinter(NullPrinter())
+        local_bears, global_bears = collect_bears(
+            [os.path.abspath("bears")],
+            ["**"],
+            [BEAR_KIND.LOCAL, BEAR_KIND.GLOBAL],
+            printer,
+            warn_if_unused_glob=False)
+        self.bears = list(itertools.chain(local_bears, global_bears))
+        self.JSONEncoder = create_json_encoder(use_relpath=False)
+
+    def test(self):
+        ERROR_STR = ("At least one bear in the \"bears\" directory is not "
+                     "JSON serializable. If you have modified any bear then "
+                     "make sure all non-serializable attributes are private. "
+                     "More information about serializable attributes can be "
+                     "found at https://docs.python.org/3.4/library/"
+                     "json.html#encoders-and-decoders")
+        try:
+            json.dumps(self.bears, cls=self.JSONEncoder)
+        except BaseException:
```

Comment:
@arafsheikh Neat. Sorry but just add one more thing. Generally it's bad practice to catch broad exceptions. But since here we need to I am thinking why not also add a small comment explaining why we are catching broad exception? What do you think?
Once done, the PR is ready :)